src/main/java/meteordevelopment/meteorclient/systems/modules/world/AirPlace.java

```diff
@@ -99,4 +105,29 @@ private void onRender(Render3DEvent event) {
 
         event.renderer.box(((BlockHitResult) hitResult).getBlockPos(), sideColor.get(), lineColor.get(), shapeMode.get(), 0);
     }
+
+    private void setHitResult() {
+        final double r = customRange.get() ? range.get() : mc.interactionManager.getReachDistance();
+        for (int i = (int) r; i > 0; i -= 1D) {
+            hitResult = mc.getCameraEntity().raycast(Math.min(r, i), 0, false);
+            if (hitResult instanceof BlockHitResult && isValid(((BlockHitResult) hitResult).getBlockPos())) return;
+        }
+        hitResult = null;
+    }
+
+    private boolean isValid(BlockPos pos) {
+        if (pos.equals(mc.player.getBlockPos())) {
+            return false;
+        }
+        if (!onlyColliding.get()) {
+            return true;
+        }
+        for (Direction dir : Direction.values()) {
+            assert mc.world != null;
+            if (mc.world.getBlockState(pos.add(dir.getOffsetX(), dir.getOffsetY(), dir.getOffsetZ())).getMaterial() != Material.AIR) {
+                return true;
+            }
+        }
+        return false;
```

Comment:
Can't this be simplified to use `BlockUtils.getPlaceSide()` ?

Like,
```
return BlockUtils.getPlaceSide(pos) != null;
```

Reply:
yes